src/components/GetInTouch.astro

```diff
@@ -48,7 +48,7 @@
 			></path>
 		</svg>
 		<span>
-			The Collab Lab is on Twitter
+			The Collab Lab were on Twitter
 			<a href="https://twitter.com/_collab_lab">@_collab_lab</a>.
```

Comment:
We should be consistent with whether we're singular or plural. I think singular sounds better looking at the Mastadon link.